vela-templates/gen_addons.go

```diff
@@ -66,7 +66,7 @@ const (
 )
 
 // DefaultEnableAddons is default enabled addons
-var DefaultEnableAddons = []string{"fluxcd", "kruise", "terraform"}
+var DefaultEnableAddons = []string{}
```

Comment:
Do we still need this feature? Why not remove the relevant code as well?

Reply:
Yes, we need it.